clar2wasm/src/words/equal.rs

```diff
@@ -9,6 +9,28 @@ use crate::wasm_generator::{
     clar2wasm_ty, drop_value, ArgumentsExt, GeneratorError, SequenceElementType, WasmGenerator,
 };
 
+trait EqCompatible {
+    fn compatible(&self, other: &Self) -> bool;
+}
```

Comment:
Creating an entire trait for a function that will be used only once at a single place seems a little bit over-engineered.

Reply:
fixed